lib/survey_gizmo/resource.rb

```diff
@@ -62,7 +62,9 @@ def all(conditions = {})
 
       # Retrieve a single resource.  See usage comment on .all
       def first(conditions = {})
-        new(conditions.merge(Connection.get(create_route(:get, conditions)).body['data']))
+        data = Connection.get(create_route(:get, conditions)).body['data']
+        data = {} if data.empty?
```

Comment:
can we get a comment, something like "Surveygizmo sometimes returns an empty array when it shouldn't"